botocore/utils.py

```diff
@@ -389,6 +389,20 @@ def parse_to_aware_datetime(value):
     return datetime_obj
 
 
+def datetime2timestamp(dt, default_timezone=tzutc()):
+    """Calculate the timestamp based on the given datetime instance.
+
+    The optional default_timezone is only used when dt is a naive datetime.
```

Comment:
It would be good to get complete docstrings for the parameters instead of just that sentence. So something like:
```
:type dt: ...
:param dt: ...
:type default_timezone: ...
:param default_timezone: ...
```